getting_hired/preparing_for_job_search/what_you_can_do_to_prepare.md

```diff
@@ -16,7 +16,7 @@ You may be uncomfortable "selling yourself" but that's exactly what you'll have
 
 All along the way, you should keep building things.  The work you've done will be crucial to demonstrating your technical skill and getting hired.  Employers want to see that you can build and ship code and that you make good decisions when confronted with engineering trade-offs.  All of this stems from conversations about your portfolio of work.
 
-The portfolio is also important for another thing that isn't mentioned much -- social proof.  If you've been a part of larger or group projects, that's almost like showing that you've worked somewhere before.  It makes the employer comfortable that other people have worked with you before so they can too.
+The portfolio is also important for another thing that isn't mentioned much -- social proof.  If you've been a part of larger or group projects, that's almost like showing that you've worked somewhere before.  It makes the employer comfortable that other people have worked with you before so they can too. Speaking of group projects, participating in the open source community is a great way to showcase your skills and collaborate with other developers. If you're looking for your first open source project to contribute to, start with TOP! We're always looking for ways to our curriculum, main site, and odin-bot v2, so check out our [repositories on GitHub](https://github.com/TheOdinProject) to get started. While you're at it, check out this [Dev.to article](https://dev.to/opensauced/how-i-got-hired-contributing-to-open-source-projects-546i) to learn you can leverage open source contributions in your job search.
```

Comment:
Overall the new wording is great, but I think it would flow better as a new paragraph. In addition, I think "We're always looking for ways to our curriculum, main site, and odin-bot v2," is missing the word "improve" (or something like that).

Reply:
Please don't mark conversations resolved. It is the job for the person that asked for the change to make sure that the changed happened & completely fulfilled the request.

I said that I believe this would flow better as a new paragraph. Did you just miss this suggestion, or do you have a reason that you think it should stay one paragraph?

Reply:
Oh, I misread your request @rlmoser99. Sorry :(

Reply:
Made the change @rlmoser99